python/handlebarrz/pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
   "structlog>=25.2.0",
 ]
 description = "Handlebars library for Python based on handlebars-rust."
-name = "dotprompt_handlebars"
+name = "dotpromptz_handlebars"
```

Comment:
Is this correct? Should it not be the same as the name used in dependencies above?